src/commands/set-config.js

```diff
@@ -0,0 +1,22 @@
+const path = require('path')
+const CourtConfigManager = require('../models/modules/CourtConfigManager')
+
+const command = 'set-config'
+const describe = 'Change Court config'
+
+const builder = {
+  input: { alias: 'i', describe: 'Court config JS file', type: 'string', default: './data/input/court-addresses' },
+}
+
+const handlerAsync = async (environment, { network, input }) => {
+  const config = require(path.resolve(process.cwd(), input))[network]
```

Comment:
Not sure if we do this for other commands, but I think it'd be nice to log which config file is being used :)

Reply:
we don't but I agree it would be nice :)